test/base.py

```diff
@@ -339,6 +339,11 @@ def select_all_streams_and_fields(conn_id, catalogs, select_all_fields: bool = T
             connections.select_catalog_and_fields_via_metadata(
                 conn_id, catalog, schema, [], non_selected_properties)
 
+    def parseunix_date(self, date_value):
```

Comment:
I think parse_unix_date is more readable.

Reply:
Updated